src/components/charts/chartHelper.ts

```diff
@@ -28,19 +32,36 @@ ChartJS.register(
 // See https://date-fns.org/v2.27.0/docs/format
 export const CHART_DATE_FORMAT = "MMM do hh:mm";
 
-export const GLOBAL_CHART_OPTIONS: ChartOptions<"line"> = {
+export const GLOBAL_LINE_CHART_OPTIONS: ChartOptions<"line"> = {
```

Comment:
Not sure this is worth caring about, but this seems to work and keep you from having to create two identical constants.

```
export const GLOBAL_CHART_OPTIONS: ChartOptions<"line" | "bar"> = {
  responsive: true,
  interaction: {
    mode: "index",
    intersect: false,
  },
};
```

Reply:
Nice find! Any way to cut out some code is good imo. I've condensed into one variable.

Reply:
I was also able to use the same trick for the two `getChartOptions` functions and combine them into 1 as well!